x-pack/plugins/cross_cluster_replication/server/routes/api/follower_index.js

```diff
@@ -100,6 +107,37 @@ export const registerFollowerIndexRoutes = (server) => {
     },
   });
 
+  /**
+   * Edit a follower index
+   */
+  server.route({
+    path: `${API_BASE_PATH}/follower_indices/{id}`,
+    method: 'PUT',
+    config: {
+      pre: [ licensePreRouting ]
+    },
+    handler: async (request) => {
+      const callWithRequest = callWithRequestFactory(server, request);
+      const { id: _id } = request.params;
+      const body = removeEmptyFields(serializeAdvancedSettings(request.payload));
+
+      try {
+        /**
+         * We need to first pause the follower and then resume it passing the advanced settings
+         */
```

Comment:
Really minor nit, but my OCD kicks in when we mix our comment styles. 😅 Could we stick with `//` for inline comments?